frontend/src/app/app.module.ts

```diff
@@ -24,9 +24,11 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
     ToastrModule.forRoot(),
     FontAwesomeModule,
   ],
+
   providers: [
-    BrowserAnimationsModule,
+    BrowserAnimationsModule
```

Comment:
Do you use Prettier?

Reply:
Yes, I already set it to work "Format in save mode"